Error creating database from model: January 24, 2025 15:31:48
'(' was never closed (create_db_models.py, line 166)